sdk/spring/CHANGELOG.md

```diff
@@ -37,6 +51,12 @@ This section includes changes in `spring-cloud-azure-autoconfigure` module.
 - Fix the Service Bus JMS autoconfiguration logic error [#29313](https://github.com/Azure/azure-sdk-for-java/pull/29313).
 - Fix the authority host of azure identity client not configured bug [#29398](https://github.com/Azure/azure-sdk-for-java/issues/29398). 
 
+### Spring Cloud Azure Service
+This section includes changes in `spring-cloud-azure-service` module.
+
+#### Dependency Updates
+- Upgrade `azure-cosmos` to 4.32.0. [#29679](https://github.com/Azure/azure-sdk-for-java/pull/29679)
```

Comment:
Same here, we are listing some of our most outstanding modules here, maybe logical projects. No need to iterate each artifact.